config/routes.rb

```diff
@@ -1,17 +1,17 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  scope '/api/compliance/' do
+  scope "#{ENV['PATH_PREFIX']}/#{ENV['APP_NAME']}" do
     resources :profiles, only: [:index, :show]
     resources :rule_results, only: [:index]
     resources :systems, only: [:index]
     resources :rules, only: [:index, :show]
-    mount Rswag::Api::Engine => '/api-docs'
-    mount Rswag::Ui::Engine => '/api-docs'
-    mount ActionCable.server => '/cable'
+    mount Rswag::Api::Engine => '/'
+    mount Rswag::Ui::Engine => '/'
```

Comment:
Could you give some context here? Our docs are moving to `/api/compliance/` from `/api/compliance/api-docs/`?

Reply:
Correct, if you visit the root of the API - it displays the Swagger docs.